### DISCUSSION TITLE
Getting "Unhandled endpoint" error while trying test scenarios on appium for iOS and Timeout error for Android

### DISCUSSION THREAD

**Discussion by SharveParihar** [appium/appium#20495] [Q&A]
I am trying to run my test scenarios on iOS as well as on Android but all the tests are failing on the login step. I will share screenshots of my error as well - 
[iOS] 

<img width="1471" alt="image (1)" src="https://github.com/user-attachments/assets/b0907357-dadc-49f8-9665-1f09e775c08e">

[Android]

<img width="830" alt="Screenshot 2024-08-22 at 5 31 09 PM" src="https://github.com/user-attachments/assets/b0a31066-7417-400b-a3eb-db885873b67f">

I am running the appium server GUI in the background and my appium version is 1.22.3, also I have tried upgrading my version and running it on 2.5.1 and 2.5.4 but the error still persists.

P.S. If you think I have missed anything and something else is required please ask for it.

**Comment by mykola-mokhnach** [comment]
https://github.com/appium/appium-uiautomator2-driver?tab=readme-ov-file#a-gesture-like-scroll-or-swipe-does-not-have-any-effect--it-is-unclear-how-to-do-it

> Make sure you don't use deprecated JWP touch actions APIs. They have been removed from the UIA2 driver since version 3.
